Nos/Controller/RefreshController.swift

```diff
@@ -19,9 +19,9 @@ import Foundation
 @Observable @MainActor class DefaultRefreshController: RefreshController {
     var shouldRefresh: Bool
 
-    var lastRefreshDate: Date?
+    var lastRefreshDate: Date
 
-    init(shouldRefresh: Bool = false, lastRefreshDate: Date? = nil) {
+    init(shouldRefresh: Bool = false, lastRefreshDate: Date = .now) {
```

Comment:
I know you are adding comments in the other PRs, so I'm not asking to add doc comments here.

Reply:
Yeah, I'll add docs in my next PR. It's almost ready.

Reply:
Actually, I just realized it is ready, and there are docs in it. #1389